src/storage/constants.ts

```diff
@@ -25,6 +25,22 @@ export enum StateDiffs {
   SELECTED_RESOURCE = "diffs.selectedResource",
 }
 
+/** Single enum to hold all of the keys the extension uses
```

Comment:
Gonna eventually get all of the above individual / distinct enum families holding individual workspace storage keys migrated to be keys within this one new enum to rule them all. No reason to have them all in separate enums when they're all used in the same ultimate namespace, keys within workspace storage.

But one step at a time, and start the new one in the right place.